go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/kong/go-kong v0.13.0
 	github.com/lithammer/dedent v1.1.0
 	github.com/mitchellh/mapstructure v1.3.1
+	github.com/pkg/errors v0.9.1
```

Comment:
Please do not introduce this dependency. We removed it purposefully in the past. Please use Go 1.13's error wrapping where required.

Reply:
Discussed out of band: https://godoc.org/errors#As should provide equivalent functionality using a standard Go library available as of 1.13.

Reply:
Done. Decided not to wrap this error.